internal/iam/command_rbac_role_binding_list.go

```diff
@@ -88,28 +88,28 @@ func (c *roleBindingCommand) newListCommand() *cobra.Command {
 		)
 	}
 
-	cmd.Flags().String("principal", "", "Principal whose role bindings should be listed.")
-	cmd.Flags().Bool("current-user", false, "Show role bindings belonging to the current user.")
-	cmd.Flags().String("role", "", "List role bindings under a specific role given to a principal. Or if no principal is specified, list principals with the role.")
+	cmd.Flags().String("principal", "", "Principal ID, which limits role bindings to this principal. If unspecified, list all principals and role bindings.")
+	cmd.Flags().Bool("current-user", false, "List role bindings assigned to the current user.")
+	cmd.Flags().String("role", "", "Predefined role assigned to `--principal`. If `--principal` is unspecified, list all principals assigned the role.")
```

Comment:
There is a feature (really a bug, in my opinion) in the `pflag` library that doesn't allow us to use backticks in flag descriptions.
```suggestion
	cmd.Flags().String("role", "", `Predefined role assigned to "--principal". If "--principal" is unspecified, list all principals assigned the role.`)
```